wled00/data/index.js

```diff
@@ -2571,6 +2578,24 @@ function setBalance(b)
 	requestJson(obj);
 }
 
+function rmtTgl(ip,i) {
+	event.preventDefault();
+	event.stopPropagation();
+	fetch(`http://${ip}/win&T=2`, {method: 'get'})
```

Comment:
We could consider `T=2&NN` so that really only that one instance is toggled (no sync). On the other hand, it might be more intuitive to have this power button do exactly what the native one in the device UI itself does. Which would you prefer?

Reply:
IMO sync should be respected if enabled.